src/net.h

```diff
@@ -145,6 +145,8 @@ class CConnman
         std::vector<std::string> vSeedNodes;
         std::vector<CSubNet> vWhitelistedRange;
         std::vector<CService> vBinds, vWhiteBinds;
+        bool m_use_addrman_outgoing = true;
```

Comment:
I'd rather not do this, as everything else here is set to null, false, or uninitialized. A value of true is unexpected.